.eslintignore

```diff
@@ -4,4 +4,6 @@
 /examples
 /mithril.js
 /mithril.min.js
+/stream.js
```

Comment:
@spacejack (quoting from [your most recent reply](https://github.com/MithrilJS/mithril.js/pull/1832#issuecomment-304364168))

> Currently, using require/import you get the pre-bundled mithril.js. My attempt was to create a similar stream.js bundle.

Oh okay. That makes sense (although that begs the question of why it's still in a subdirectory - the main export is an unbundled UMD).

But for the purposes of this review, just ignore this specific request (the original one).